src/peft/tuners/tuners_utils.py

```diff
@@ -494,6 +494,24 @@ def _unloading_checks(self, adapter_names: Optional[list[str]]):
         if is_modules_to_save_available and len(adapters_to_consider) > 1:
             raise ValueError("Cannot unload multiple adapters that specify `modules_to_save`.")
 
+    @staticmethod
+    def get_model_config(model, default={"model_type": "custom"}):
```

Comment:
Good idea. Let's add a docstring that explains what this returns, mentioning that the config will be converted to a dict and that custom models will just get a dummy config.

Reply:
Addressed the comments 👍